Dockerfile

```diff
@@ -189,14 +189,14 @@ USER root
 ### GO
 
 # Install Go
-ARG GOLANG_VERSION=1.18.4
+ARG GOLANG_VERSION=1.19
```

Comment:
For major releases, the `.0` should be omitted for package download URLs, otherwise it 404's.